### ISSUE TITLE
Eggs should not be possible to close once opened.

### ISSUE THREAD

**Issue by SpadesNeil** [SpadesNeil/VOREstation#647]
I've seen players abusing eggs to continually spam going back into the egg and closing the egg after it has already been hatched open.

It's not supposed to work that way.

In fact, using lockers for eggs is really hacky and bad to begin with.

**Comment by Cameron653** [comment]
Yeah. I choose a locker originally as it was the simplest thing I could think of to get it to work, as it'd let prey struggle out of the egg, and let preds take the prey out of the egg.

Also, it's made so that if you use a welder on it, then it'll get rid of the egg to prevent that, and as a way to get rid of already used eggs.
